launch_snakemake.py

```diff
@@ -302,6 +318,8 @@ def analysis_main(args, output, runnormal=False, normalname=False, normalfastqs=
             if args.normalsample:
             # these functions are only executed if snakemake workflow has finished successfully
                 yearly_stats(args.tumorsample, args.normalsample)
+                if args.hg38ref:
```

Comment:
We need a flag for not doing alissa upload when testing things, like nocompress. Since there is a cost for creating patients/samples

Reply:
solved in cf88586